pyiron/atomistics/job/atomistic.py

```diff
@@ -473,6 +473,10 @@ def trajectory(
 
         """
         cells = self.output.cells
+        if len(self.output.indices) != 0:
+            indices = self.output.indices
+        else:
+            indices = self.get("output/generic/indices")
```

Comment:
Does this work with old calculation? once which were calculated before this update?

Reply:
Good catch. It does now; if the indices aren't there I just cover by using the structure's indices at each frame.